### PR TITLE
Implement Javascript in Mobile Features

### PR DESCRIPTION
**Pull request by RitaDee**
### In this milestone, we implemented the following:

- Add the JavaScript + HTML + CSS linter.
- Made the mobile menu appears when the user clicks (or taps) the hamburger button.
- Made users click (or tap) the close (X) button and the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -426,6 +437,25 @@ span {
...
+
+.nav-item.active {
+  display: none;
+}
+
+.nav-item a {
+  color: #fff;
+  font-size: 32px;
+  margin-top: 50%;
+}
```
Comment (sja-thedude) [inline style.css]: - [ ] Kindly make changes for the mobile menu from [here](https://www.loom.com/share/0752d20c4e5b4ebebad7f0b8f18b8561).

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @RitaDee, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Correct gitflow
✔️ Professional Readme

## Required 
- [ ] Kindly remove the broken microverse logo from readme as it's not required.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -23,12 +23,15 @@
...
             <li class="nav-item">
               <a class="link" href="#about">About</a>
+              <img class="seperator" src="/images/List Item/Width Bottom.png" alt="" srcset="">
             </li>
             <li class="nav-item">
               <a class="link" href="#contact">Contact</a>
+              <img class="seperator" src="/images/List Item/Width Bottom.png" alt="" srcset="">
             </li>
             <li class="nav-item last-menu-item">
               <a class="link" href="#msg">
```
Comment (omarsalem7) [inline index.html]: - [ ] As stated before by the previous reviewer, kindly make sure to add a separator like in the Figma design.

- [ ] [OPTIONAL] I suggest using the correct font type and weight like the Figma design.

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @RitaDee 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Kindly fix the broken license link in your readme.

- [ ] [OPTIONAL] It will be better if you add a `future features` section in your readme by including 1-3 potential sentences about what you will don in the future.
 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -439,7 +439,7 @@ span {
 
 .nav-item {
   margin-bottom: 20px;
-  padding-top: 55px;
+  padding-top: 12px;
   padding-left: 14px;
   position: relative;
```
Comment (GedeonTS) [inline style.css:444]: - [ ] In my opinion, you should stick to the design as suggested by the previous reviewer by bringing back the separators to the nav items.
- your design:
your desing
- Figma design:
figma

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @RitaDee, this is @GedeonTS your reviewer, 

Great work on making most of the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights :tophat: :+1: 

- Linter checks are green  :+1:
- Good use of Github flow :+1:
- The popup is working fine :+1: 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 
- [ OPTIONAL ] Please make sure you write more meaningful commit messages, describing exactly what change you just made.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (RitaDee) [reply style.css:444]: I guess you are looking at the wrong link. Please check again.

Comment (GedeonTS) [reply style.css:444]: Hi @RitaDee, if you have any doubt you might be right, please don't hesitate to ask for a second opinion by using this [link](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hi @RitaDee  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted the `mobile menu `at  the place of the details `popup window`, Kindly submit the project in a correct place in your dashboard.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css (hunk 3 above)
Comment (prolajumokeoni) [reply style.css:444]: > I guess you are looking at the wrong link. Please check again.

I see all the changes requested has been implemented 

Screenshot_587

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (RitaDee) [reply style.css:444]: Thank you for the check and confirmation Jumoke.

**Review by nicupop729 — COMMENTED** [review]
Hello @RitaDee  👋,

_This is an additional review as a result of the second opinion request._

I am glad to see that your project is approved.
Congrats! :clap: 

PS: next time when you request a Second Opinion Review, please wait for it before asking for a regular review again; it may take longer to receive it, but because you were right, it could have saved you a review attempt. 

Keep rocking 🚀!

**Comment by RitaDee** [comment]
Thanks for the feedback!

**Pull request merged**